orchestrator.Jenkinsfile

```diff
@@ -360,6 +361,8 @@ global.connector_service_tests.version=${versions['tests/connector-service-tests
 global.connector_service_tests.dir=${versions['tests/connector-service-tests'] == env.BRANCH_NAME ? 'pr/' : 'develop/'}
 global.metadata_service_tests.version=${versions['tests/metadata-service-tests']}
 global.metadata_service_tests.dir=${versions['tests/metadata-service-tests'] == env.BRANCH_NAME ? 'pr/' : 'develop/'}
+global.remote-environment-controller-tests.version=${versions['tests/remote-environment-controller-tests']}
```

Comment:
Please use underscores instead of dashes, dashes are not allowed as YAML keys.
See all other components in this list for reference.

Reply:
Done